### PR TITLE
Fixed permissions so an admin users can't create an account anymore

### PR DESCRIPTION
**Pull request by glower**
Only users with SuperAdmin rights are allowed to create new accounts. This means that the `Account` tab in the admin UI is not visible to all other users anymore (same logic as for the `User` tab).

### CODE REVIEW

File: service/account/handlers_accounts_test.go [hunk 1]
```diff
@@ -153,25 +153,31 @@ func (s *AccountSuite) TestCreateGetUpdateAccountHandlers(c *check.C) {
 
 	account := datastore.Account{ID: 2, AuthorityID: "vendor", ResellerAPI: false}
 	acc, _ := json.Marshal(account)
+	accBrokenJSON := []byte(`{"user"`)
 
 	tests := []AccountTest{
```
Comment (doismellburning) [inline service/account/handlers_accounts_test.go]: This is not especially easy or pleasant to review and I'm surprised by the sheer number of changes here, including ones that don't seem obviously relevant

Thoughts on splitting it up into single-change commits / adding some more clarity in some other way?

**Comment by doismellburning** [comment]
The Go changes certainly look good (though I struggle with the test changes)!

The JS seems reasonable at a glance but I don't really JS...

File: service/account/handlers_accounts_test.go (hunk 1 above)
Comment (glower) [reply service/account/handlers_accounts_test.go]: I added some missing test cases, but you are right, I will remove all not necessary test cases from this PR (but yes, the way the tests are structured makes it hard to fix/review)

**Comment by glower** [comment]
The changes in the frontend code are removing the navigation tab `Accounts` from the UI if the user does not have `SuperUser` status

**Comment by glower** [comment]
> The Go changes certainly look good (though I struggle with the test changes)!
Removed all not necessary changes in the tests. For the better overview of the changes, I need to reformat all the tests to not to use the short form for the map. But this change is just too large 

> The JS seems reasonable at a glance but I don't really JS...
I will ask for the extra FE code review

File: webapp-admin/src/components/AccountEdit.js [hunk 2]
```diff
@@ -37,7 +37,7 @@ class AccountEdit extends Component {
     getAccount(accountId) {
         Accounts.get(accountId).then((response) => {
             var data = JSON.parse(response.body);
-
+
```
Comment (Lukewh) [inline webapp-admin/src/components/AccountEdit.js]: Cheeky spaces

File: service/account/handlers_accounts_test.go (hunk 1 above)
Comment (doismellburning) [reply service/account/handlers_accounts_test.go]: Thanks! Definitely agree that these tests _generally_ aren't easy to review, but it's much clearer now you've dropped the extraneous ones cheers!

**Pull request merged**